junifer/markers/utils.py

```diff
@@ -88,7 +99,21 @@ def _ets(bold_ts: np.ndarray) -> np.ndarray:
     # indices of unique edges (lower triangle)
     u, v = np.tril_indices(n_roi, k=-1)
     # Compute the ETS
-    return timeseries[:, u] * timeseries[:, v]
+    ets = timeseries[:, u] * timeseries[:, v]
+    # Obtain the corresponding edge labels if specified else return
+    if roi_names is None:
+        return ets
+    else:
+        if len(roi_names) != n_roi:
```

Comment:
These lines are not covered by unit tests.

Reply:
I will add a test.